components/admin_console/system_users/system_users.jsx

```diff
@@ -116,7 +115,12 @@ export default class SystemUsers extends React.Component {
         } else if (teamId === SearchUserTeamFilter.NO_TEAM) {
             loadProfilesWithoutTeam(0, Constants.PROFILE_CHUNK_SIZE, this.loadComplete);
         } else {
-            loadProfilesAndTeamMembers(0, Constants.PROFILE_CHUNK_SIZE, teamId, this.loadComplete);
+            this.props.actions.loadProfilesAndTeamMembers(0, Constants.PROFILE_CHUNK_SIZE, teamId).then(({data}) => {
```

Comment:
If you going to use `async` you should use `await` here, if not, you can leave the function without `async` and handle here the promise. I prefer the former option.

Reply:
Right, I totally missed that.  I'll use the async/await.